common/src/main/java/com/nearsoft/fgaribay/mgmt/config/BrokerConfig.java

```diff
@@ -0,0 +1,76 @@
+package com.nearsoft.fgaribay.mgmt.config;
+
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:common.properties")
+@ComponentScan("com.nearsoft.fgaribay.mgmt.config")
```

Comment:
By default, `@ComponentScan` uses the same package of the class that declares this annotation. You can safely remove the hardcoded package.